vars/runBenchmarkTestScript.groovy

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+/** Library to execute benchmark-test using opensearch-benchmark and opensearch-cluster-cdk
+ *
+ * @param Map args = [:] args A map of the following parameters
+ * @param args.bundleManifest <required> - OpenSearch bundle manifest url.
+ * @param args.insecure <required> - Force the security of the cluster to be disabled, default is false.
+ * @param args.workload <required> - Name of the workload that OpenSearch Benchmark should run, default is nyc_taxis.
+ * @param args.singleNode <required> - Create single node OS cluster, default is true.
+ * @param args.minDistribution <required> - Use min distribution of OpenSearch for cluster, default is false.
+ * @param args.useFiftyPercentHeap <required> - Use 50 percent of physical memory as heap, default is false.
```

Comment:
Match the actual param? `use50percentHeap`

Reply:
Right, done!